script.js

```diff
@@ -1,12 +1,370 @@
-//You can edit ALL of the code here
+let allEpisodes;
+let searchResult = document.createElement("div");
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  const allShows = getAllShows();
```

Comment:
You've assigned the result of `getAllShows` to the variable on line 5 but only pass it into the function on line 8. Couldn't you also pass it into line 7 and 8 instead of calling `getAllShows` again on lines 78 and 122?